lib/s3storage.php

```diff
@@ -85,10 +85,11 @@ protected function init() {
 				return;
 			}
 			// force content length header on empty body
-			$request->setHeader('Content-Length', 0);
+			$request->setHeader('Content-Length', "0");
```

Comment:
`phan` was complaining that `setHeader` paramater 2 must be a string. IMO these header things are strings anyway, even if they sometimes look like integers.